client/packages/invoices/src/InboundShipment/DetailView/modals/InboundLineEdit/TabTables.tsx

```diff
@@ -146,10 +134,15 @@ export const QuantityTableComponent = ({
     getColumnLookupWithOverrides('packSize', {
       Cell: PackSizeEntryCell<DraftInboundLine>,
       setter: patch => {
-        setPackRoundingMessage?.('');
```

Comment:
should this have been removed? probably makes more sense here than on the shipped pack size right?